src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -1319,6 +1319,8 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 /* Home Info Card */
 
+"Home_AppInformationCard_Title" = "App-Information";
```

Comment:
Why is this here? ^^

Reply:
Reverted back the string, it is now contained in Omar's PR.